R/predictPreeclampsia.R

```diff
@@ -0,0 +1,91 @@
+#' @title predictPreeclampsia
+#'
+#' @description Uses 45 CpGs to predict early preeclampsia (PE delivered before or at 34 weeks of gestation)
+#' on placental DNA methylation microarray data.
+#'
+#' @details Assigns the class labels "early-PE" or "normotensive" to each sample
+#' and returns a class probability.
+#'
+#' It is recommended that users apply beta-mixture quantile normalization (BMIQ) to their data
+#' prior to prediction. This was the normalization method used on the training data.
+#'
+#' @param betas matrix or array of methylation values on the beta scale (0, 1),
+#' where the variables are arranged in rows, and samples in columns.
+#'
+#' @return produces a list with components detailed in the `mixOmics::predict` R documentation
+#'
+#' @examples
+#'
+#' To predict early preeclampsia on 450k/850k samples
+#'
+#' Load data
+#' data(peBetas)
+#' predictPreeclampsia(peBetas, dist = "max.dist")
+#' 
+#' @import mixOmics
+#' @import ExperimentHub
```

Comment:
I don't think we need to import the whole packages, can we remove these?

Reply:
I wasn't sure how to fix the error that the devtools check was throwing other than by adding these as imports, is there a better way?

Reply:
if it's fixed by importing the whole package, then likely there is some mixOmics function that ins't being called without double colons e.g. should be `dplyr::slice` instead of just `slice`.

Reply:
I still think taking dependency on mixomics and experimenthub is overkill -> particularly mixomics seems like a large package and my worry is that it will 1) if mixomics has dependencies itself then many more subdependencies will be created

I can take a look